core/controlplane/config/templates/cloud-config-controller

```diff
@@ -2711,14 +2743,31 @@ write_files:
     content: {{.AssetsConfig.EtcdTrustedCA}}
 {{ end }}
 
+  - path: /etc/kubernetes/proxy-kubeconfig.yaml
+    content: |
+        apiVersion: v1
+        kind: Config
+        clusters:
+        - name: local
+          cluster:
+            server: http://127.0.0.1:8080
+        users:
+        - name: kubelet
+        contexts:
+        - context:
+            cluster: local
+            user: kubelet
+          name: kubelet-context
+        current-context: kubelet-context
+
   - path: /etc/kubernetes/controller-kubeconfig.yaml
     content: |
         apiVersion: v1
         kind: Config
         clusters:
         - name: local
           cluster:
-            server: http://localhost:8080
+            server: http://127.0.0.1:8080
```

Comment:
ipv6 issues?

Reply:
I don't know about that, only that there are several other places using `127.0.0.1` instead of `localhost`.

What do you propose? To change everything to `localhost`?

Reply:
Just asking why this change. Maybe you hit some problem with `localhost` resolved to ipv6 ` ::1` .

Reply:
With this change, the only place where `localhost` is being used is in the `kube-node-label.service` unit, but I personally don't have a preference. Will undo this change and deal with the 127.0.0.1 vs localhost issue in another PR, if needed.

Reply:
just to be clear, I am fine with either, was curious to ask why that change, but not verbose enough in my question :)